static/src/components/PlayerDisplay.js

```diff
@@ -117,109 +101,118 @@ export class PlayerDisplay extends Component {
-                                  <TableRowColumn>
-                                    <Card
-                                        expanded={this.state.expanded}
-                                        onExpandChange={this.handleExpandChange}
-                                    >
-                                        <CardHeader
-                                            id="CardHeader"
-                                            title={pName}
-                                            subtitle={boardData.wonder}
-                                            avatar={<Avatar src={`dist/images/cards/age${game.cards[0].age}.png`} size={55} />}
-                                            actAsExpander={true}
-                                            showExpandableButton={true}
-                                        />
-                                        <CardText id="CardText" style={{ paddingBottom: 0, paddingTop: 0 }}>
-                                            <div>
-                                                <Table>
-                                                    <TableBody displayRowCheckbox={false} >
-                                                        <TableRow selectable={false}>
-                                                            <TableRowColumn
-                                                                style={inventorycustomColumnStyle}
-                                                            >
-                                                                <List style={ListStyle}>
-                                                                    <Inventory item="wood" amount={boardData.wood + boardData.extra_wood} />
-                                                                    <Inventory item="brick" amount={boardData.brick + boardData.extra_brick} />
-                                                                    <Inventory item="ore" amount={boardData.ore + boardData.extra_ore} />
-                                                                    <Inventory item="stone" amount={boardData.stone + boardData.extra_stone} />
-                                                                    <Inventory item="glass" amount={boardData.glass + boardData.extra_glass} />
-                                                                    <Inventory item="paper" amount={boardData.paper + boardData.extra_paper} />
-                                                                    <Inventory item="cloth" amount={boardData.cloth + boardData.extra_cloth} />
-                                                                </List>
-                                                            </TableRowColumn>
-                                                            <TableRowColumn>
-                                                                <CardActions>
-                                                                    { !homeWonder ?
```

Comment:
Did you mean to delete this?

Reply:
Pretty sure i haven't deleted any bit of this render, just got rid of a couple syntax error don't know why its saying i changed the entire table.
Have fixed up those accidental deletions and console log though.

Reply:
Could be an issue with git but merging this PR will remove this functionality in PlayerDisplay.

Reply:
This doesn't look like it's been deleted.
lines 135-222 have just been tabbed correctly by the looks
Below the red section is the correctly tabbed green section